README.md

```diff
@@ -17,11 +17,12 @@ python3 -m pip install sootty
 To use, run:
 
 ```bash
-sootty waveform.vcd > image.svg
+sootty -f "waveform.vcd" > image.svg
```

Comment:
I added the `-f` flag because when the user wants to reload a query, he might not want to specify the filename and other things again, and so having the filename as a positional argument was a problem, as it was not necessary when using the reload functionality.